charts/testkube-enterprise/values.yaml

```diff
@@ -238,6 +238,27 @@ testkube-worker-service:
   image:
     repository: testkubeenterprise/testkube-enterprise-worker-service
     tag: 1.9.0-rc-e5167f6a
+
+testkube-logs-service:
```

Comment:
If you want to include logs service into Enterprise chart, you also need to add it to `Chart.yaml`